### PR TITLE
testing/peervpn: new aport

### PR DESCRIPTION
**Pull request by mjuenema**
Adding peervpn (https://peervpn.net) package - second attempt ;-)

### CODE REVIEW

**Comment by jirutka** [comment]
The previous attempt: #600

File: testing/peervpn/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=peervpn
+pkgver=0.044
+pkgrel=0
+pkgdesc="PeerVPN builds virtual ethernet networks between multiple computers"
+url="https://peervpn.net"
+arch="all"
+license="GPL3"
+depends="openssl zlib"
+makedepends="$depends openssl-dev zlib-dev linux-headers"
```
Comment (fabled) [inline testing/peervpn/APKBUILD]: Does this work with libressl in depends and libressl-dev in makedepends instead?
We generally don't want new packages that depend on openssl.

Comment (mjuenema) [inline testing/peervpn/APKBUILD]: I don't know. I have to test it first. i didn't know that you are phasing out Openssl.

Comment (jirutka) [inline testing/peervpn/APKBUILD]: We’ve migrated to LibreSSL in Alpine v3.5. Are you preparing new aports on edge or at least v3.5?

Comment (mjuenema) [inline testing/peervpn/APKBUILD]: I guess i should aim for the most recent release. Would that be with libressl? Is it possible to create a package that could work with either? This is my first package So i might need some guidance.

Comment (mjuenema) [inline testing/peervpn/APKBUILD]: I checked and it works with libressl and libressl-dev. Do you want to update the merge request or shall I make the change in the branch and push it?

Comment (jirutka) [inline testing/peervpn/APKBUILD]: Please make the change and push it into your branch `adding-peervpn`.

**Pull request closed without merging**